dev.docker-compose.yml

```diff
@@ -120,7 +123,9 @@ services:
         tty: true
 
     proxy:
-        image: nginx:1.15-alpine
+        build:
```

Comment:
@you can probably comment this out. we moved away from having devs doing local docker builds by default.
Its painful for pople with slow conectivity